src/rules/noUnusedVariableRule.ts

```diff
@@ -40,6 +41,17 @@ class NoUnusedVariablesWalker extends Lint.RuleWalker {
         this.languageService = languageService;
     }
 
+    public visitBindingElement(node: ts.BindingElement) {
+        var isSingleVariable = node.name.kind === ts.SyntaxKind.Identifier;
+
+        if (isSingleVariable && !this.skipBindingElement) {
+            var variableIdentifier = <ts.Identifier> node.name;
+            this.validateReferencesForVariable(variableIdentifier.text, variableIdentifier.getStart());
+        }
+
+        super.visitBindingElement(node);
+    }
+
     public visitImportDeclaration(node: ts.ImportDeclaration): void {
         if (!this.hasModifier(node.modifiers, ts.SyntaxKind.ExportKeyword)) {
```

Comment:
yep, that is implemented in a separate PR: https://github.com/palantir/tslint/pull/411